src/main/java/graphql/testrunner/service/PrepareEnvironmentService.java

```diff
@@ -0,0 +1,45 @@
+package graphql.testrunner.service;
+
+import java.util.List;
+
+import graphql.testrunner.util.TestRunnerException;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import static java.util.Arrays.asList;
+
+import static graphql.testrunner.service.CommandExecutorService.BUILD_GRAPHQL_JAR;
+import static graphql.testrunner.service.CommandExecutorService.BUILD_GRAPHQL_JMH_JAR;
+import static graphql.testrunner.service.CommandExecutorService.GRAPHQL_DIR;
+
+@Service
+public class PrepareEnvironmentService {
+
+    @Autowired
+    private GitService gitService;
+
+    @Autowired
+    private CommandExecutorService commandExecutorService;
+
+    /**
+     * Clones the graphql-java project and prepares the jars required for tests.
+     *
+     * @param commitHash
+     * @throws TestRunnerException
+     */
+    public void prepareJars(String commitHash) throws TestRunnerException {
+        gitService.pullCode(commitHash);
+        buildGraphqlJar();
```

Comment:
Do we need to build graphql-java jar or building only jmh jar is enough?(tried without this step and it worked!)

Reply:
if you don't need it, don't include it!

Reply:
removed.